prediction_market_agent/agents/think_thoroughly_agent/deploy.py

```diff
@@ -34,7 +34,7 @@ class DeployableThinkThoroughlyAgent(DeployableThinkThoroughlyAgentBase):
 
 class DeployableThinkThoroughlyProphetResearchAgent(DeployableThinkThoroughlyAgentBase):
     agent_class = ThinkThoroughlyWithPredictionProphetResearch
-    model: str = "gpt-3.5-turbo-0125"
+    model: str = "gpt-4-turbo-2024-04-09"
```

Comment:
According to the latest Dune stats, prophet with gpt4turbo is one of the best agents, so switching this one to gpt4turbo as well.